templates/zerver/integrations.html

```diff
@@ -1801,6 +1801,54 @@
 
     </div>
 
+    <div id="papertrail" class="integration-instructions">
+
+
+      <p>
+        Zulip supports integration with Papertrail as a
+        <a href="http://www.perforce.com/perforce/doc.current/manuals/p4sag/chapter.scripting.html">Webhook</a>
+        that fires upon finding a certain log. To do this:
+      </p>
+
+      <p>
+        First, create the stream you'd like to use for Papertrail
+        notifications, and subscribe all interested parties to this stream. We
+        recommend the stream name be <code>papertrail</code>. After create a bot named
+        <code>Papertrail</code> and note the API key.
+      </p>
+
+      <p>
+        Next, log into your Papertrail Account and browse through your logs.
+        Search for logs you want to get alerts for and press the <code>Save Search</code>,
+        though be sure to press the <code>Save & Setup an alert button</code>
```

Comment:
I think you need to tell that after clicking `Save Search`, a modal window will open. Click `Save & Setup an alert` button on that modal window.